packages/tessellate-server/src/nconf.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import fs from 'fs'
+import path from 'path'
+import nconf from 'nconf'
+import yaml from 'js-yaml'
+
+function fileExists(file: string): boolean {
+  try {
+    fs.statSync(file)
+  } catch (e) {
+    if (e.code == 'ENOENT') return false
+    else throw e
+  }
+  return true
+}
+
+function readFile(file: string, parser: (s: string) => Object): Object {
+  try {
+    return parser(fs.readFileSync(file, 'utf8'))
+  } catch (e) {
+    console.error(`Unable to read ${file}`)
+    return {}
+  }
+}
+
+function readJSONFile(file: string): Object {
```

Comment:
For consistency we should not capitalize JSON, so `readJSONFile()` -> `readJsonFile().